jenkins/catapult/compilers/msvc-2019.yaml

```diff
@@ -0,0 +1,9 @@
+c: msvc
+cpp: bar
```

Comment:
bar ?

Reply:
et me check.  I think I got these from the other branch.  Let see if there something better